app/model/lib/submission_process.py

```diff
@@ -608,17 +616,7 @@ def _get_expected_column_names(submission_form):
         else:
             raise ValueError(f"Unexpected technique subjectType: {technique.subjectType}")
 
-    expected_sheets = {
-        'Growth data per community':  community_columns,
-        'Growth data per strain':     strain_columns,
-        'Growth data per metabolite': metabolite_columns,
-    }
-
-    for key in list(expected_sheets.keys()):
-        if len(expected_sheets[key]) == 0:
-            del expected_sheets[key]
-
-    return expected_sheets
+    return community_columns, strain_columns, metabolite_columns
```

Comment:
Instead of grouping the required columns into specifically named sheets, we now only need to validate that the columns are present anywhere in the uploaded spreadsheet.